tests/integration_tests/security/test_vulnerabilities.py

```diff
@@ -280,9 +280,8 @@ def check_vulnerabilities_files_on_guest(microvm, template=None):
     for vuln_file in vuln_files:
         filename = os.path.basename(vuln_file)
         if filename in exceptions:
-            cmd = f"grep '{exceptions[filename]}' {vuln_file}"
-            ecode, stdout, stderr = microvm.ssh.run(cmd)
-            assert ecode == 0, f"stdout:\n{stdout}\nstderr:\n{stderr}\n"
+            _, stdout, _ = microvm.ssh.run(f"cat {vuln_file}")
```

Comment:
I assume this prints the file as well for context, as part of the local variables?

Reply:
yes, it shows several context when it fails, like this:
```
_ test_vulnerabilities_files_on_restored_guest_with_template[static_C3-vmlinux-5.10.186-ubuntu-22.04.squashfs] _
integration_tests/security/test_vulnerabilities.py:358: in test_vulnerabilities_files_on_restored_guest_with_template
    check_vulnerabilities_files_on_guest(dst_vm, template)
        dst_vm     = <Microvm id=4bb8f07b-4dc9-4299-b21f-382309d3bf93>
        microvm    = <Microvm id=80365b09-da96-4b7d-9ad0-01d130f57284>
        microvm_factory = <framework.microvm.MicroVMFactory object at 0x7f4b1ce93430>
        microvm_with_template = (<Microvm id=80365b09-da96-4b7d-9ad0-01d130f57284>, 'C3')
        snapshot   = Snapshot(vmstate=PosixPath('/srv/jailer/firecracker/80365b09-da96-4b7d-9ad0-01d130f57284/root/vmstate'), mem=PosixPath...64/ubuntu-22.04.squashfs')}, ssh_key=PosixPath('/srv/img/x86_64/ubuntu-22.04.id_rsa'), snapshot_type=SnapshotType.FULL)
        template   = 'C3'
integration_tests/security/test_vulnerabilities.py:284: in check_vulnerabilities_files_on_guest
    assert exceptions[filename] in stdout
E   AssertionError: assert 'Vulnerable: Clear CPU buffers attempted, no microcode' in 'Unknown: No mitigations\n'
        _          = ''
        cmd        = 'grep Vulnerable /sys/devices/system/cpu/vulnerabilities/itlb_multihit'
        ecode      = 1
        exceptions = {'mmio_stale_data': 'Vulnerable: Clear CPU buffers attempted, no microcode'}
        filename   = 'mmio_stale_data'
        microvm    = <Microvm id=4bb8f07b-4dc9-4299-b21f-382309d3bf93>
        stderr     = ''
        stdout     = 'Unknown: No mitigations\n'
        template   = 'C3'
        vuln_dir   = '/sys/devices/system/cpu/vulnerabilities'
        vuln_file  = '/sys/devices/system/cpu/vulnerabilities/mmio_stale_data'
        vuln_files = ['/sys/devices/system/cpu/vulnerabilities/spectre_v2', '/sys/devices/system/cpu/vulnerabilities/itlb_multihit', '/sys/...mds', '/sys/devices/system/cpu/vulnerabilities/l1tf', '/sys/devices/system/cpu/vulnerabilities/spec_store_bypass', ...]
```